src/main/java/com/mitchej123/hodgepodge/mixins/Mixins.java

```diff
@@ -624,7 +624,7 @@ public enum Mixins implements IMixins {
             .addMixinClasses("fml.MixinGuiModList", "fml.MixinGuiSlotModList", "fml.MixinGuiScrollingList")
             .setApplyIf(
                     () -> TweaksConfig.betterModList
-                            && !classExists("com.enderio.core.common.transform.EnderCoreTransformerClient"))
+                            && !classExists("com/enderio/core/common/transform/EnderCoreTransformerClient.class"))
```

Comment:
Looks good, but I'd prefer to keep the classname rather than the class file path.
Is there any concern you changed it?

Reply:
Otherwise, the function name should be renamed to like `doesResourceExists` imo